src/http.zig

```diff
@@ -379,15 +379,57 @@ fn NewHTTPContext(comptime ssl: bool) type {
 
         pub fn init(this: *@This()) !void {
             if (comptime ssl) {
-                const opts: uws.us_bun_socket_context_options_t = .{
+                var opts: uws.us_bun_socket_context_options_t = .{
                     // we request the cert so we load root certs and can verify it
                     .request_cert = 1,
                     // we manually abort the connection if the hostname doesn't match
                     .reject_unauthorized = 0,
                 };
+
+                var extra_cas: ?std.ArrayList([*c]const u8) = null;
+
+                blk: {
+                    if (std.process.getEnvVarOwned(default_allocator, "NODE_EXTRA_CA_CERTS")) |extra_ca_certs_path| {
```

Comment:
This should use something in `DotEnv.Loader` and be cached there. We will need to add support for this in several places instead of only in the http client.

Reply:
Okey, that was the main thing I was unsure about. I initially looked at going that route, but it ended up being kind of complicated with the current architecture of how/when the HTTPThread is initialized.

But then I'll move this there instead and I will have to change the initialization of the `HTTPThread` to depend on `DotEnv.Loader`. I do agree it's also better option devex wise.

If you have any ideas on how to best get the ca file contents into the context of the https context initialization, I'd happily take some pointers as well :sweat_smile: (the first/only way I saw how to do this involved passing the dotenv context around alot)